libgadget/petaio.c

```diff
@@ -481,7 +481,10 @@ petaio_read_header_internal(BigFile * bf) {
         endrun(0, "Failed to read required cosmology from IC header\n");
     }
 
-    /* Fall back to use a**2 * dx/dt if UsePeculiarVelocity is not set in IC */
+    /* If UsePeculiarVelocity = 1 then snapshots save to the velocity field the physical peculiar velocity.
+     * If UsePeculiarVelocity = 0 then snapshots save a * physical peculiar velocity in snapshots
```

Comment:
I settled to explicitly write down whenever these are mentioned the exact formula 
- a dx / dt for 'physical peculiar velocity', 
- a^2 dx / dt for conjugate momentum (per unit mass internal integration quantity, BlueTides-i/ii) and
- a^{1/2} dx / dt for Gadget-1 velocity.

I think the formula makes conversion clear and avoids confusion.

Reply:
That is a good idea. Just to explicitly check that I have this right: x is comoving distance in the above, right?